server/src/main/java/org/elasticsearch/action/admin/indices/stats/CommonStats.java

```diff
@@ -218,6 +225,9 @@ public CommonStats(IndicesQueryCache indicesQueryCache, IndexShard indexShard, C
                     case Bulk:
                         bulk = indexShard.bulkStats();
                         break;
+                    case Shards:
+                        shards = new SimpleShardStats(1);
```

Comment:
A comment about why this is set to 1 may be useful here for future code spelunkers